postBuild

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-python3 -m pip install --index-url https://pypi.cs.uni-tuebingen.de/simple/ pyopenms-nightly
+python3 -m pip install --index-url https://pypi.cs.uni-tuebingen.de/simple/ pyopenms
```

Comment:
Is this even necessary? I thought binder installs based on the environment/requirements?

Reply:
no clue... maybe related to caching the base environment without postBuild stuff?